custom_components/better_thermostat/climate.py

```diff
@@ -255,7 +258,7 @@ def _async_startup(*_):
 			_async_startup()
 		else:
 			self.hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, _async_startup)
-		
```

Comment:
Please keep the format. Don't remove tabs from empty code lines where it's just the code indentation